test/Altinn.App.Api.Tests/Utils/PrincipalUtil.cs

```diff
@@ -30,6 +36,23 @@ public static ClaimsPrincipal GetUserPrincipal(int userId, int authenticationLev
             ClaimsPrincipal principal = new ClaimsPrincipal(identity);
             return principal;
         }
+        
+        public static ClaimsPrincipal GetUserPrincipal(int userId, int partyId, int authenticationLevel = 2)
+        {
+            List<Claim> claims = new List<Claim>();
+            string issuer = "www.altinn.no";
+            claims.Add(new Claim(ClaimTypes.NameIdentifier, userId.ToString(), ClaimValueTypes.String, issuer));
+            claims.Add(new Claim(AltinnCoreClaimTypes.UserId, userId.ToString(), ClaimValueTypes.String, issuer));
+            claims.Add(new Claim(AltinnCoreClaimTypes.UserName, "UserOne", ClaimValueTypes.String, issuer));
+            claims.Add(new Claim(AltinnCoreClaimTypes.PartyID, partyId.ToString(), ClaimValueTypes.Integer32, issuer));
```

Comment:
Hvorfor setter vi `PartyID` claim til `UserId` i de eksisterende testene? Det er da faktisk feil, og kan dekke over ganske kjedelige bugs.

Prøvde å se over eksisterende steder dette er brukt, og det ser ut som en veldig enkel job å legge på en default parameter `partyId = null` i de orginale metodene (`GetToken` og `GetUserPrincipal`) og endre denne linjen til
```suggestion
            if (partyId > 0)
            {
                claims.Add(new Claim(AltinnCoreClaimTypes.PartyID, partyId.ToString(), ClaimValueTypes.Integer32, issuer));
            }
```